package.json

```diff
@@ -112,7 +115,7 @@
     "webpack": "5.75.0"
   },
   "engines": {
-    "node": ">= 14"
+    "node": ">= 18"
```

Comment:
Why is this different from the normal ember blueprint?

Reply:
Got carried to far by the wind of change... :eagle:

Reply:
```
Run yarn install
yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error @adfinis/semantic-release-config@4.0.0: The engine "node" is incompatible with this module. Expected version ">= 1[8](https://github.com/projectcaluma/ember-emeis/pull/657/checks#step:5:9)". Got "16.20.0"
```